lib/game_logic.rb

```diff
@@ -0,0 +1,101 @@
+require('rainbow')
+# rubocop:disable Style/GlobalVars,Metrics/MethodLength
+$board = [1, 2, 3, 4, 5, 6, 7, 8, 9]
+
+$winner = false
+
+class Drawer
+  def draw_board
+    board_sign = $board
+    puts("\t#{board_sign[0]} | #{board_sign[1]} | #{board_sign[2]} ")
+    puts "\t---------"
+    puts("\t#{board_sign[3]} | #{board_sign[4]} | #{board_sign[5]} ")
+    puts "\t---------"
+    puts("\t#{board_sign[6]} | #{board_sign[7]} | #{board_sign[8]} ")
+    puts "\n"
+  end
+
+  def winner?(sign)
+    board_sign = $board
+    arr = [[0, 1, 2], [3, 4, 5], [6, 7, 8], [0, 3, 6], [1, 4, 7], [2, 5, 8], [0, 4, 8], [2, 4, 6]]
+    arr.each do |element|
+      $winner = element.all? { |index| board_sign[index] == sign }
+      return true if $winner
+    end
+
+    $winner
+  end
+end
+
+class Player
+  attr_accessor :moves
+  attr_reader :sign
+
+  def initialize(sign, name)
+    @sign = sign
+    @moves = []
+    @name = name
+  end
+
+  def make_move
+    loop do
+      puts Rainbow("Enter the #{@name} move").blue.bright
+      move = gets.chomp.to_i
+      is_valid = choose_number(move)
+      unless is_valid
+        puts Rainbow("\nWrong input, try again!\n").red
```

Comment:
No `puts/print` and `get.chomp` outside `main.rb` file. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/only_main_print.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))